src/libbsdl/include/BSDL/config.h

```diff
@@ -0,0 +1,123 @@
+// Copyright Contributors to the Open Shading Language project.
+// SPDX-License-Identifier: BSD-3-Clause
+// https://github.com/AcademySoftwareFoundation/OpenShadingLanguage
+
+
+#pragma once
+
+#ifndef BSDL_NS
+#    define BSDL_NS bsdl
+#endif
+
+#define BSDL_ENTER_NAMESPACE namespace BSDL_NS {
+#define BSDL_LEAVE_NAMESPACE }
+
+#ifndef BSDL_INLINE_METHOD
+#    define BSDL_INLINE_METHOD inline
+#endif
+#ifndef BSDL_INLINE
+#    define BSDL_INLINE inline
+#endif
+#ifndef BSDL_DECL
+#    define BSDL_DECL
+#endif
+#ifndef BSDL_UNROLL
+#    define BSDL_UNROLL() _Pragma("unroll")
+#endif
+
+#include <cassert>
+#include <cmath>
+
+#ifndef M_PI
+#    define M_PI 3.1415926535897932
+#endif
+
+BSDL_ENTER_NAMESPACE
+
+BSDL_DECL constexpr float EPSILON   = 1e-4f;
+BSDL_DECL constexpr float PI        = float(M_PI);
+BSDL_DECL constexpr float ONEOVERPI = 1 / PI;
+BSDL_DECL constexpr float ALMOSTONE
+    = 0.999999940395355224609375f;  // Max float  < 1.0f
+BSDL_DECL constexpr float FLOAT_MIN
+    = 1.17549435e-38f;  // Minimum float normal value
+BSDL_DECL constexpr float SQRT2   = 1.4142135623730951f;
+BSDL_DECL constexpr float BIG     = 1e12f;
+BSDL_DECL constexpr float PDF_MIN = 1e-6f;
+
+struct BSDLDefaultConfig {
+    struct Fast {
+        static BSDL_INLINE_METHOD float cosf(float x) { return std::cos(x); }
+        static BSDL_INLINE_METHOD float sinf(float x) { return std::sin(x); }
+        static BSDL_INLINE_METHOD void sincosf(float x, float* s, float* c)
+        {
+            *s = std::sin(x);
+            *c = std::cos(x);
+        }
```

Comment:
Aren't these the slow versions?

Reply:
Yes, the defaults. See that we override them in bsdl_config.h for testrender